### PR TITLE
add caution note

### PR DESCRIPTION
**Pull request by kingthorin**
Also make "license" vs "licence" consistent in help.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,11 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+CAUTION - While we do review all scripts to ensure they don't do anything obviously malicious, you should still review them and use them with caution.
```
Comment (ricekot) [inline CHANGELOG.md:6]: Will this show up anywhere in the ZAP UI?

Comment (thc202) [reply CHANGELOG.md:6]: No.

Comment (thc202) [reply CHANGELOG.md:6]: Maybe this one should be moved to the help page.

Comment (kingthorin) [reply CHANGELOG.md:6]: Okay,will do that

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**